vcxagency-node/src/logging/logger-builder.js

```diff
@@ -20,6 +20,7 @@ const winston = require('winston')
 const path = require('path')
 const { jsonFormatter, tryAddRequestId, tryAddEcsTaskMetadata } = require('./logger-common')
 
+console.log(`DISABLE_COLOR_LOGS= ${global.DISABLE_COLOR_LOGS}`)
```

Comment:
I am not sure about this log, if we are to leave it let's definitely make it `debug` and `DISABLE_COLOR_LOGS=${global.DISABLE_COLOR_LOGS}`

Reply:
yup this was forgotten, deleted now